add_map not declared in abigen

Looks like in a recent commit to remove tools (commit: a43f6a818e3d5718bb6b33c7fb9b57e41585b9d1) `add_map` function was removed, but not all references to it in `abigen.hpp`.  This is resulting in a `abigen.hpp:269:16: error: ‘add_map’ was not declared in this scope
  269 |                add_map(type);`

--- a/tools/include/eosio/abigen.hpp
+++ b/tools/include/eosio/abigen.hpp
@@ -265,8 +265,6 @@ namespace eosio { namespace cdt {
             else if (is_template_specialization(type, {"vector", "set", "deque", "list", "optional", "binary_extension", "ignore"})) {
                add_type(std::get<clang::QualType>(get_template_argument(type)));
             }
-            else if (is_template_specialization(type, {"map"}))
-               add_map(type);
             else if (is_template_specialization(type, {"pair"}))
                add_pair(type);
             else if (is_template_specialization(type, {"tuple"}))

